test/include/RAJA_test-atomic-types.hpp

```diff
@@ -32,4 +32,11 @@ using AtomicDataTypeList =
 #endif
               double >;
 
+using AtomicDataUnsignedTypeList =
```

Comment:
I think we'll need to guard this to run only when Desul is activated. CUDA and HIP cannot do unsigned atomics.

Reply:
Hmm, I was under the impression that CUDA did support unsigned atomics from my brief work helping with desul. If that isn't the case then a macro guard definitely makes sense.

Reply:
Ah, you're right, it should work with `unsigned int` and `unsigned long long int`. I think I had put the unsigned cases into the exhaustive tests before.

Reply:
Ahh cool, is that just for CUDA or does HIP also support unsigned atomics? I'm not too familiar with HIP.

Reply:
They should both support them.  Hip is trying to be cuda compatible, and in both cases it should just work for everything (the wrapping atomic increment is special, since that supports very few targets, as are the floating point ones but for integral types it should just go).